### ISSUE TITLE
Level range

### ISSUE THREAD

**Issue by sansthesis** [chronicles-mud/chronicles#10]
I believe stock has a level cap of 60, with 1-51 being morts and 52-60 being immortals.  Should we change this?

Do we need 8 ranks of immortals?  Do we keep their privileges the same as stock?

**Comment by XTremeTennis** [comment]
I don't think we need eight ranges of immortals, four or five should do it, right? Level 1 IMM has basic commands to make sure they dont screw up, 2 gets the important commands, 3 gets commands that can really be abused, and 4 gets CP access or something along those lines.

I think lowering the level from 101 is a good idea because it allows you to focus skills better, I was thinking in the range of 70-75 as a cap for morts, but a whole host of level ranges would work.

**Comment by XTremeTennis** [comment]
scaling exp/level like most games, IDK something like

| Level | EXP TNL | MOB Kills @ 250 |
| --- | --- | --- |
| 1 | 1,000 | 4 |
| 5 | 2,000 | 8 |
| 10 | 4,500 | 18 |
| 15 | 6,078 | 24 |
| 20 | 7,757 | 31 |
| 25 | 9,900 | 40 |
| 30 | 12,635 | 51 |
| 35 | 16,125 | 65 |
| 40 | 20,581 | 82 |
| 45 | 26,267 | 105 |
| 50 | 33,524 | 134 |

this table is manual until level 11 (+250/level until 5 then +500/level until 11), then +5% from there on.
Mob kills @ 250 is how many mobs, at 250xp given each, would you have to kill to level up.
